lte/gateway/deploy/roles/magma/tasks/main.yml

```diff
@@ -134,73 +135,79 @@
+    - { src: '{{ c_build }}/sctpd/src/sctpd', path: sbin/sctpd }
+    - { src: '{{ go_build }}/envoy_controller', path: envoy_controller }
+    - { src: '{{ oai_build }}/oai_mme/mme', path: bin/mme }
   when: full_provision
 
 - name: Create symlink for DHCP helper CLI binary
-  file: src='/home/{{ ansible_user }}/magma/lte/gateway/python/dhcp_helper_cli/dhcp_helper_cli.py' path=/usr/local/bin/dhcp_helper_cli.py state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for OAI mme binary
-  file: src='{{ oai_build }}/oai_mme/mme' path=/usr/local/bin/mme state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for session_manager binary
-  file: src='{{ c_build }}/session_manager/sessiond' path=/usr/local/bin/sessiond state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for envoy controller binary
-  file: src='{{ go_build }}/envoy_controller' path=/usr/local/bin/envoy_controller state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for dpid binary
-  file: src='{{ c_build }}/dpi/dpid' path=/usr/local/bin/dpid state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for connectiond binary
-  file: src='{{ c_build }}/connection_tracker/src/connectiond' path=/usr/local/bin/connectiond state=link force=yes follow=no
-  when: full_provision
-
-- name: Create symlink for li agent binary
-  file: src='{{ c_build }}/li_agent/src/liagentd' path=/usr/local/bin/liagentd state=link force=yes follow=no
+  file:
+    src: '{{ magma_root }}/lte/gateway/python/dhcp_helper_cli/dhcp_helper_cli.py'
+    path: /usr/local/bin/dhcp_helper_cli.py
+    state: link
+    force: yes
+    follow: no
   when: full_provision
 
 - name: Create symlink for corefile collection script
-  file: src='{{ magma_root }}/lte/gateway/deploy/roles/magma/files/coredump' path=/usr/local/bin/coredump state=link force=yes follow=no
+  file:
+    src: '{{ magma_root }}/lte/gateway/deploy/roles/magma/files/coredump'
+    path: /usr/local/bin/coredump
+    state: link
+    force: yes
+    follow: no
   when: full_provision
 
-- name: Create the /var/www/local-cdn directory
-  file: path=/var/www/local-cdn state=directory
-  when: preburn
-
 - name: Create the /var/www/local-cdn/store directory
-  file: path=/var/www/local-cdn/store state=directory
+  file:
+    path: /var/www/local-cdn/store
+    state: directory
   when: preburn
 
 - name: Add Magma package directory
   become: no
   file:
     path: '{{ magma_repo }}/'
     state: directory
-  when: full_provision
+  when: preburn
 
 - name: Add Magma dependency package directory
   become: no
   file:
     path: '{{ magma_deps }}/'
     state: directory
-  when: full_provision
+  when: preburn
 
 - name: Enable IP forwarding
-  sysctl: name="net.ipv4.ip_forward" value=1 sysctl_set=yes state=present reload=yes
+  sysctl:
+    name: net.ipv4.ip_forward
+    value: '1'
```

Comment:
Nit: Are the quotes needed here?

Reply:
Okay - just seems curious given that we do not use quotes on the other integer values.